.buildkite/pipeline.yml

```diff
@@ -32,28 +29,33 @@ steps:
   #################
   - label: "🧹 Lint"
     command: |
-      .buildkite/install-dependencies.sh
-
+      bundle install
       echo "--- :rubocop: Run Rubocop"
       bundle exec rubocop
     env: *common_env
-    plugins: *common_plugins
+    plugins:
+      - docker#v3.8.0:
+          image: "ruby:2.7.4"
     agents:
-      queue: "mac"
+      queue: "default"
 
   #################
   # Danger
   #################
   - label: "⛔️ Danger"
     command: |
-      .buildkite/install-dependencies.sh
-
+      bundle install
```

Comment:
Same question as to wether this is intentional not to `brew install` during this step. If it is, worth a quick comment to indicate it was intentional and not an accidental overlook.

Reply:
I left a note where we use `brew install` directly to hopefully help clarify when it is / is not required.